projects/Mallard/package.json

```diff
@@ -34,12 +34,14 @@
     "devDependencies": {
         "@babel/core": "^7.4.4",
         "@babel/runtime": "^7.4.4",
+        "@creditkarma/thrift-server-core": "^0.13.6",
+        "@guardian/capi-ts": "^0.1.0",
         "@types/jest": "^24.0.12",
+        "@types/node": "^12.0.4",
```

Comment:
Required by `@creditkarma/thrift-server-core`, although this fact is not documented alongside other implicit peer dependencies in their [readme](https://github.com/creditkarma/thrift-server#install-dependencies)

Reply:
Note that we won't need either of these modules when the [below issue](https://github.com/guardian/editions/pull/66#discussion_r290684113) has been resolved